src/prod_components/agorize.js

```diff
@@ -195,13 +195,13 @@ function Agorize({
 				        	<Input.TextArea
 				        		name="body"
 										disabled={!authorized}
+										autosize={{minRows: 4}}
 				        		placeholder={
 				        			agoragramType !== "comment" ?
-				        				"Text (icke-obligatoriskt)\nDetta är markdown, tryck på frågetecknet för mer information. "
+				        				"Text (icke-obligatoriskt)\nDetta är react-markdown, tryck på frågetecknet för mer information. "
```

Comment:
React-markdown is the package that compiles markdown to react. There should be little difference between this compiler and other compilers.

Therefore I think it's more correct to say it's markdown. It is also how GitHub refers to it. 
<img width="209" alt="image" src="https://user-images.githubusercontent.com/28629647/68686666-43c7dd00-056c-11ea-8e24-afeb426fd43a.png">

However, this is such a small problem I don't think this needs to be fixed now. The component will eventually be deprecated in favour of [draft.js](https://github.com/facebook/draft-js).